nextroute/schema/input.go

```diff
@@ -203,6 +203,17 @@ type Stop struct {
 	ID string `json:"id,omitempty"`
 	// Location where the stop is.
 	Location Location `json:"location,omitempty"`
+	// MixingItems defines the items that are inserted or removed from the vehicle when visiting the stop.
+	MixingItems any `json:"mixing_items,omitempty"`
```

Comment:
Should this be of type `MixItem`?

Reply:
No there can be multiple items that can not be mixed on a vehicle